### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [janczizikow/sleek#56]
The user @carlsonsantana validate your site template "https://janczizikow.github.io/sleek/" and found these accessibility errors:
* Page https://janczizikow.github.io/sleek/:
  * Issue *ca7ed726-07bf-4852-90e8-7b01eb2e26fa*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > footer > div > nav > a:nth-child(3)`
    * **Context**: `<a class="social__link" target="_blank" rel="noopener noreferrer" href="https://www.instagram.com/shanlongjj"> <svg class="social__icon" view...</a>`
  * Issue *08c634a8-a2b8-4bae-9f25-80b981a9a684*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > footer > div > nav > a:nth-child(2)`
    * **Context**: `<a class="social__link" target="_blank" rel="noopener noreferrer" href="https://www.twitter.com/jan_czizikow"> <svg class="social__icon" view...</a>`
  * Issue *454c5d40-e3f0-496c-bbff-62a547f9d445*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > footer > div > nav > a:nth-child(1)`
    * **Context**: `<a class="social__link" target="_blank" rel="noopener noreferrer" href="https://github.com/janczizikow"> <svg class="social__icon" view...</a>`
  * Issue *b47e7d0a-327b-44ab-9457-25b8fa659311*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(4) > a:nth-child(3) > p`
    * **Context**: `<p>Lorem ipsum dolor sit amet, con...</p>`
  * Issue *20e6d5ca-1c67-48d9-b877-0522e71167ae*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(4) > a:nth-child(3) > div > span > time`
    * **Context**: `<time>May 20, 2016</time>`
  * Issue *95e36f1d-6f71-42b8-bcb2-434dae616292*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(3) > a:nth-child(3) > p`
    * **Context**: `<p>You’ll find this post in your _...</p>`
  * Issue *55f4e7be-cf0b-459a-aff6-f01ddb36e29c*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(3) > a:nth-child(3) > div > span > time`
    * **Context**: `<time>May 20, 2016</time>`
  * Issue *7d63ab0e-ada3-43db-b055-d21963326c57*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(3) > a:nth-child(2)`
    * **Context**: `<a href="/sleek/welcome-to-jekyll/"><figure class="post-card__thumb...</a>`
  * Issue *75a80401-8880-4545-995f-c87208de1ed7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(2) > a:nth-child(3) > p`
    * **Context**: `<p>Markdown is a way to style text...</p>`
  * Issue *3dde787b-5652-45dc-8fcb-f2b08f417201*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(2) > a:nth-child(3) > div > span > time`
    * **Context**: `<time>Oct 15, 2017</time>`
  * Issue *89a8e9ef-10b5-4305-bcec-47299c35c48e*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(1) > a:nth-child(3) > p`
    * **Context**: `<p>Getting started GitHub Pages ca...</p>`
  * Issue *91f82e6b-574c-4272-91bc-82cbad85dfe4*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.44:1. Recommendation: change text colour to #6f7879.
    * **Selector**: `html > body > main > section > div > div > div:nth-child(1) > a:nth-child(3) > div > span > time`
    * **Context**: `<time>Nov 26, 2017</time>`
  * Issue *e61b947b-ac9e-45dd-85a2-c3d72bfd4d2f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
    * **Message**: Anchor element found with a valid href attribute, but no link content has been supplied.
    * **Selector**: `html > body > header > div > div > a`
    * **Context**: `<a class="header__logo" href="/sleek/"><div class="header__logo--conta...</a>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/b36dc437-2ef9-4e36-bf67-353af3bd810a/).
